### PR TITLE
Add warning about marshmallow 2 EOL

### PR DESCRIPTION
**Pull request by sloria**
ref #1582

### CODE REVIEW

**Comment by sloria** [comment]
Image 2020-05-04 at 10 21 21 PM

File: docs/conf.py [hunk 1]
```diff
@@ -104,23 +104,16 @@
 
 # sphinx-version-warning config
 versionwarning_messages = {
-    'latest': (
-        'This document is for the development version. '
-        'For the stable version documentation, see <a href="/en/stable/">here</a>.'
-    ),
```
Comment (lafrech) [inline docs/conf.py:110]: Why remove the "latest" comment?

File: docs/conf.py [hunk 2]
```diff
@@ -104,23 +104,16 @@
...
+        'marshmallow 2 will no longer be supported as of 2020-08-18. '
+        '<a href="https://marshmallow.readthedocs.io/en/latest/upgrading.html#upgrading-to-3-0">'
+        'Update your code to use marshmallow 3</a>.'
     ),
 }
 # Show warning at top of page
 versionwarning_body_selector = "div.document"
 # For debugging locally
-# versionwarning_project_version = 'latest'
+# versionwarning_project_version = '2.x-line'
```
Comment (lafrech) [inline docs/conf.py:120]: Not clear to me what this does.

I did read https://sphinx-version-warning.readthedocs.io/en/latest/configuration.html#confval-versionwarning_project_version but I don't get it.

**Review by lafrech — APPROVED** [review]
Good to me.

Just comments / questions but you can merge.

File: docs/conf.py (hunk 1 above)
Comment (sloria) [reply docs/conf.py:110]: Ah, yeah, let's leave that. Will revert

File: docs/conf.py (hunk 2 above)
Comment (sloria) [reply docs/conf.py:120]: This is just for testing things locally. The warning banner doesn't show unless this is uncommented

**Pull request merged**